examples/tekton_pipeline_orchestration/setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I think the `setup.sh` file should only exist for examples that can be run locally, so should be removed in this case

Reply:
Ok! Shouldn't we also remove this from the kubeflow pipelines example too?

Reply:
I think the kubeflow example actually runs locally (assuming that you have the pre-requisites installed), but Tekton doesn't support that (for now)